Cargo.toml

```diff
@@ -30,6 +30,7 @@ codecov = { repository = "rust-cli/clap-verbosity-flag" }
 [dependencies]
 log = "0.4.1"
 clap = { version = "3.0", default-features = false, features = ["std", "derive"] }
+tracing = { version = "0.1", default-features = false, optional = true }
```

Comment:
How mature / stable is tracing for being a `0.1` crate?  We just went 1.0 and I don't want to repeatedly bump by taking on a potentially volatile dependency

Reply:
I'm only a very new user of `tracing`, so I don't know.  On one hand, I would guess that `tracing::Level` is too basic to be likely to disappear.  On the other hand, in hindsight, considering the crates' very different version numbers, I do think it would be more appropriate for me to ask `tracing` to add the necessary conversion code.

Reply:
>  On one hand, I would guess that tracing::Level is too basic to be likely to disappear.

My concern is not with it disappearing but that when a tracing 0.2 comes out, to support it, we'd have to release clap-verbosity-flag 2.0.  Ditto for each breaking release of tracing, regardless of whether the type looks the same.   Rust will consider `tracing::Level` from 0.1, 0.2, 0.3, etc all to be different, incompatible types.